rental.rb

```diff
@@ -0,0 +1,19 @@
+# rentak.rb
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] It would be nice to have Getters and Setters for @book and @person as required in the project requirement. The att_accessor allows for reading and writing to a file and hence, it's expected to be used for the creation and reading of a file, person, and book respectively 👍🏽

`attr_accessor :date, :book, :person`